src/js/components/PageHeader/PageHeader.js

```diff
@@ -8,13 +8,19 @@ import { Grid } from '../Grid';
 import { Paragraph } from '../Paragraph';
 import { ResponsiveContext } from '../../contexts/ResponsiveContext';
 
+const sizeStyle = (size, style, theme) =>
+  size && theme.pageHeader.size[size]?.[style]
```

Comment:
This seems a bit fragile for cases like `pad: 0`. How about:
`(size && theme.pageHeader.size[size]?.[style]) ?? theme.pageHeader[style]`
or maybe, to not require replicating all properties:
`{ ...theme.pageHeader[style], ...((size && theme.pageHeader.size[size]?.[style]) ?? theme.pageHeader[style]) }`

Reply:
Done.